android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -169,6 +207,11 @@ public void sendToNotificationCentre(Bundle bundle) {
 
             String title = bundle.getString("title");
             String notificationType = bundle.getString("notification_type");
+
+            if(this.shouldIgnoreNotification(bundle)) {
```

Comment:
You have already got the notification (and perhaps also the group id) and parsed them into `int`, why not just pass these values into `this.shouldIgnoreNotification(...)` to avoid having to do it again?